src/System.Security.AccessControl/src/System/Security/AccessControl/SecurityDescriptor.cs

```diff
@@ -305,7 +305,7 @@ public void GetBinaryForm(byte[] binaryForm, int offset)
 
             binaryForm[offset + 0] = Revision;
             binaryForm[offset + 1] = rmControl;
-            binaryForm[offset + 2] = (byte)((int)materializedControlFlags >> 0);
+            binaryForm[offset + 2] = unchecked((byte)((int)materializedControlFlags >> 0));
             binaryForm[offset + 3] = (byte)((int)materializedControlFlags >> 8);
```

Comment:
I don't think so. ControlFlags is a flags enum with the highest defined value being 0x8000.
https://github.com/dotnet/corefx/blob/master/src/System.Security.AccessControl/src/System/Security/AccessControl/SecurityDescriptor.cs#L22-L43